backend/src/main/resources/application-cloud.yaml

```diff
@@ -18,7 +18,7 @@ simple-report:
     - neil.s.sharma@omb.eop.gov
     - wsullivan@cdc.gov
     - jeremy.a.zitomer@omb.eop.gov
-    - eapau@cdc.gov
+    - nicholas.scialli@usds.dhs.gov
```

Comment:
Oh, we should probably delete this file￼. (No longer used.)

Reply:
removed in https://github.com/CDCgov/prime-simplereport/pull/789. The PR could use a review

Reply:
Oh, I somehow thought that had already been merged. Will jump on over there, yes. 🙂